toast/toast.css

```diff
@@ -0,0 +1,42 @@
+@import url(/style.css);
+.viewport {
+    height: auto ;
+}
+/*toast*/
+.toast-text {
+    margin: 1rem;
+
+}
+
+.design-success {
+    width: 1rem;
+    height: auto;
+    background-color: var(--GREEN);
+
+}
+
+.toast {
+    display: flex;
+    background-color: rgb(255, 255, 255);
+    font-size: 2rem;
+    margin: .3rem;
+    height: 10rem;
+
+}
+
+.design-dismiss {
+    width: 1rem;
+    height: auto;
+    background-color: var(--RED);
+    
+}
+
+.design-warning {
+    width: 1rem;
+    height: auto;
+    background-color: rgba(255, 255, 57, 0.781);
+}
+.fas{
+    color: green;
+    font-size: 2rem;
+}
```

Comment:
change the icon color in different toast types, green doesn't match with error

Reply:
sure thanks pankaj